spec/models/early_career_payments/award_amount_calculator_spec.rb

```diff
@@ -0,0 +1,354 @@
+require "rails_helper"
+
+describe EarlyCareerPayments::AwardAmountCalculator do
+  let(:base_school) { instance_double("School", eligible_for_early_career_payments?: true, eligible_for_early_career_payments_as_uplift?: false) }
```

Comment:
Why not the class? `instance_double(School)`?

Reply:
I didn't like the factories